aws/src/test/java/org/apache/iceberg/aws/s3/signer/TestS3RestSigner.java

```diff
@@ -71,11 +73,10 @@ public class TestS3RestSigner {
 
   private static final Region REGION = Region.US_WEST_2;
   private static final String BUCKET = "iceberg-s3-signer-test";
+  private static final MinIOContainer MINIO_CONTAINER = MinioHelper.createContainer();
   static final AwsCredentialsProvider CREDENTIALS_PROVIDER =
       StaticCredentialsProvider.create(
-          AwsBasicCredentials.create("accessKeyId", "secretAccessKey"));
-  private static final MinioContainer MINIO_CONTAINER =
-      new MinioContainer(CREDENTIALS_PROVIDER.resolveCredentials());
+          AwsBasicCredentials.create(MINIO_CONTAINER.getUserName(), MINIO_CONTAINER.getPassword()));
```

Comment:
this isn't correctly setting accessKey/secretAccessKey. The old `MinioContainer` was setting multiple env vars for this and also configured virtual-host-style addressing, which we should continue to do

Reply:
1) I have restored virtual-host-style addressing (restored MINIO_DOMAIN) 

2) MINIO_ACCESS_KEY  is deprecated in favor of  MINIO_ROOT_USER

3) MINIO_SECRET_KEY is deprecated in favor of MINIO_ROOT_PASSWORD

source: https://min.io/docs/minio/linux/reference/minio-server/settings/deprecated.html